lib/requests/get-account-info-request.js

```diff
@@ -0,0 +1,115 @@
+'use strict'
+
+const ACLChecker = require('../acl-checker')
+const AuthRequest = require('./auth-request')
+const { actualSize } = require('../utils')
+const $rdf = require('rdflib')
+const HTTPError = require('../http-error')
+const LRU = require('lru-cache')
+
+const PIM = $rdf.Namespace('http://www.w3.org/ns/pim/space#')
+const SOLID = $rdf.Namespace('http://www.w3.org/ns/solid/terms#')
+
+const cache = new LRU({
+  max: 10000, // is this a reasonable amount to hold?
+  length: (n, key) => {
+    const keyType = typeof key
+    let length = 1
+    switch (keyType) {
+      case 'object':
+        length = Object.keys(key).length
+        break
+      case 'string':
+        length = key.length
+        break
+    }
+    return n * 2 + length
+  },
+  maxAge: 1000 * 60 * 60 // hold the value for one hour
+})
+
+class GetAccountInfoRequest extends AuthRequest {
+  static async get (req, res) {
+    const contentType = req.get('content-type') || 'text/turtle'
+    const body = await getBody(req, contentType)
+    if (body.status && body.status >= 400) {
+      return res.status(body.status).send(body.message)
+    }
+    return res.status(200).set('Content-Type', contentType).send(body)
+  }
+}
+
+async function getBody (req, contentType) {
+  const userId = req.session.userId
+  const cacheKey = `${userId}-${contentType}`
```

Comment:
I would not include `contentType` in the cache key.
Instead, we could just cache the graph, and re-serialize the graph for every request.
This would solve the problem where the whole quota calculation needs to be re-done if someone asks for a different content type.
Also, this would make cache invalidation a bit easier.